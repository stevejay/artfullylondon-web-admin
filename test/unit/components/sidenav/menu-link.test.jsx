import React from 'react'
import { shallow } from 'enzyme'
import _ from 'lodash'

import MenuLink from '_src/components/sidenav/menu-link'

it('should render correctly', () => {
  const wrapper = shallow(
    <MenuLink path='/the/path' label='The Label' onClick={_.noop} />
  )

  expect(wrapper).toMatchSnapshot()
})
