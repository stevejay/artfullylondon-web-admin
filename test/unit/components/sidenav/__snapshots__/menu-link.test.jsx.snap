// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Link
  onClick={[Function]}
  replace={false}
  styleName="link"
  to="/the/path"
>
  The Label
</Link>
`;
